### PR TITLE
Feature/SC-2710 - Implementing time zones

### PR DESCRIPTION
**Pull request by DanielRieckhof**
# Description
select timezone when creating school

## Links to Tickets or other pull requests

- https://ticketsystem.hpi-schul-cloud.org/browse/SC-2710
- Superhero Dashboard: https://github.com/hpi-schul-cloud/superhero-dashboard/pull/86

## Changes

Added timezone property to school form

## New Repos, NPM pakages or vendor scripts
    "moment-timezone": "^0.5.31",
to get all timezones

## Screenshots of UI changes
image
image

### CODE REVIEW

**Review by mwolowyk-cap — CHANGES_REQUESTED** [review]
I like your implementation. 
Please write the offset like this (UTC +02:00). It is more understandable.

**Review by mwolowyk-cap — CHANGES_REQUESTED** [review]
You have a wrong sorting for parts of the hour. Would be great if it could be improved.
F.e 
- 14:00
- 14:15
- 13:00

**Comment by mwolowyk-cap** [comment]
The time zone information is not loaded when you click to edit the school.

File: views/schools/forms/add-school.hbs [hunk 1]
```diff
@@ -26,6 +26,14 @@
         {{/each}}
     </select>
 </div>
+<div class="form-group">
+    <label class="control-label" for="storageProvider">Time zone</label>
+    <select name="timeZones" type="text">
```
Comment (mwolowyk-cap) [inline views/schools/forms/add-school.hbs:31]: like in the database
```suggestion
    <select name="timezone" type="text">
```

File: controllers/schools.js [hunk 2]
```diff
@@ -7,9 +7,32 @@ const express = require('express');
...
+    rhours = rhours > 0 ? `+${rhours}` : rhours;
+    minutes = minutes === 0 ? '00' : minutes;
+    return `(UTC ${rhours}:${minutes})`;
+}
+let countryCodes = moment.tz.countries();
+countryCodes = countryCodes.map((item) => {
+    return moment.tz.zonesForCountry(item, true);
+});
+countryCodes = [].concat.apply([], countryCodes);
+countryCodes.sort((a, b) => parseFloat(a.offset) - parseFloat(b.offset));
```
Comment (mwolowyk-cap) [inline controllers/schools.js:27]: Please improve sorting for partial hours.

**Review by mwolowyk-cap — CHANGES_REQUESTED** [review]

**Comment by mwolowyk-cap** [comment]
Please add timezone to the schools overview

image

File: views/schools/forms/add-school.hbs (hunk 1 above)
Comment (ABialoskorski) [reply views/schools/forms/add-school.hbs:31]: done ;)

File: controllers/schools.js (hunk 2 above)
Comment (ABialoskorski) [reply controllers/schools.js:27]: done ;)

**Review by mwolowyk-cap — APPROVED** [review]

**Pull request merged**